src/tools/extras/spellfailure_data/SpellFailure.cpp

```diff
@@ -74,22 +75,24 @@ int find_string_in_buffer(char * str, size_t str_len, char * buf, size_t buf_len
 
 int main(int argc, char* argv[])
 {
-    FILE * in = fopen( Executable, "rb");
-    FILE * out = fopen( OutputFile, "w");
-
-    if( in == NULL ){
-        printf("ERROR: Couldn't open %s for reading!\n", Executable );
-        printf("Exiting.\n");
-        fclose(in);
-        flose(out);
+    FILE* in = std::fopen( Executable, "rb");
+    FILE* out = std::fopen( OutputFile, "w");
+
+    if (in == nullptr)
+    {
+        std::cout << "ERROR: Couldn't open %s for reading!\n";
+        std::cout << "Exiting.\n";
+        std::fclose(in);
+        std::fclose(out);
```

Comment:
Not sure if flose (C) can be replaced with std::fclose

Reply:
> close() closes a file descriptor, so that it no longer refers to any file and may be reused. Any record locks (see fcntl(2)) held on the file it was associated with, and owned by the process, are removed (regardless of the file descriptor that was used to obtain the lock).